src/main/java/ee/tuleva/onboarding/conversion/UserConversionService.java

```diff
@@ -182,11 +183,9 @@ private boolean isTransfersComplete(
   }
 
   private boolean isConvertedFundManager(Exchange exchange) {
-    return exchange
-        .getTargetFund()
-        .getFundManager()
-        .getName()
-        .equalsIgnoreCase(CONVERTED_FUND_MANAGER_NAME);
+    FundDto targetFund = exchange.getTargetFund();
+    return targetFund != null
+        && targetFund.getFundManagerName().equalsIgnoreCase(CONVERTED_FUND_MANAGER_NAME);
```

Comment:
1 option would be to move this logic into the `FundDto` class as well. I.e. so we could do something like:
`targetFund.isConverted()`